app/controllers/admin/jobs_controller.rb

```diff
@@ -28,6 +28,6 @@ def resource
   private
 
   def job_params
-    params[:job].permit(:title, :description, :published_at)
+    params.require(:job).permit(:title, :description, :company, :published_at)
```

Comment:
So `company` was never here. Meaning old Rails was either letting it in or `company` was never being set for the records?

Reply:
Gonna guess the former since we use the company name in the job titles.